xikolo-ios/Layouts/de.lproj/TabSettings.strings

```diff
@@ -1,28 +1,18 @@
-
-/* Class = "UILabel"; text = "[build]"; ObjectID = "BYP-h5-ZvT"; */
-"BYP-h5-ZvT.text" = "[build]";
-
 /* Class = "UITabBarItem"; title = "Settings"; ObjectID = "KhH-tO-TnH"; */
 "KhH-tO-TnH.title" = "Einstellungen";
 
-/* Class = "UILabel"; text = "[email]"; ObjectID = "PoF-ih-nyj"; */
-"PoF-ih-nyj.text" = " ";
+/* Class = "UILabel"; text = "Preload Content"; ObjectID = "Lvj-Rt-mif"; */
+"Lvj-Rt-mif.text" = "Inhalte vorladen";
```

Comment:
I'm not sure wether this is sufficiently clear to the user - what is content? Are videos content? Will videos get preloaded over cellular (like Facebook and instagram do it when this option is activated)? Maybe we need more explanation of some kind (dialog, alert from the bottom, explanation in place) here.